eggplant/membership/models/profile.py

```diff
@@ -8,6 +8,7 @@ class UserProfile(models.Model):
     MALE = 'm'
     FEMALE = 'f'
     SEX_CHOICES = (
```

Comment:
SEX ? :D

 Option 5:

    Female
    Genderqueer/Androgynous
    Intersex
    Male
    Transgender
    Transsexual
    Cross-dresser
    FTM (female-to-male)
    MTF (male-to-female)
    Other (please specify)

http://www.hrc.org/resources/entry/collecting-transgender-inclusive-gender-data-in-workplace-and-other-surveys